.github/workflows/auto-release.yml

```
@@ -288,9 +288,40 @@ jobs:
+          # trusted publisher. Fix path (one-time):
+          #   https://www.npmjs.com/package/glin-profanity/access
+          #   → Publishing access → Add trusted publisher → GitHub Actions:
+          #       Organization: glincker
+          #       Repository:   glin-profanity
+          #       Workflow:     auto-release.yml
+          #       Environment:  npm-publish
+          # See docs/RELEASING.md for the full walkthrough.
+          if ! npm publish --tag "$NPM_TAG" --provenance --access public; then
+            cat >&2 <<'EOF'
+
+=============================================================
```

actionlint (1.7.12)

[error] could not parse as YAML: could not find expected ':'

(syntax-check)